plugins/generic/googleAnalytics/settingsForm.tpl

```diff
@@ -32,16 +32,41 @@
 	</tr>
 	<tr valign="top">
 		<td width="20%" class="label">{fieldLabel name="trackingCode-urchin" required="true" key="plugins.generic.googleAnalytics.manager.settings.trackingCode"}</td>
-		<td width="80%" class="value"><input type="radio" name="trackingCode" id="trackingCode-urchin" value="urchin" {if $trackingCode eq "urchin" || $trackingCode eq ""}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.urchin"}</td>
-	</tr>
-	<tr valign="top">
-		<td width="20%" class="label">&nbsp;</td>
-		<td width="80%" class="value"><input type="radio" name="trackingCode" id="trackingCode-ga" value="ga" {if $trackingCode eq "ga"}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.ga"}</td>
+		<td width="80%" class="value">
+			<div><input type="radio" name="trackingCode" id="trackingCode-urchin" value="urchin" {if $trackingCode eq "urchin" || $trackingCode eq ""}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.urchin"}</div>
+			<div><input type="radio" name="trackingCode" id="trackingCode-ga" value="ga" {if $trackingCode eq "ga"}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.ga"}</div>
+			<div><input type="radio" name="trackingCode" id="trackingCode-analytics" value="analytics" {if $trackingCode eq "analytics"}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.analytics"}</div>
+		</td>
 	</tr>
+	{if $siteAdmin}
 	<tr valign="top">
-		<td width="20%" class="label">&nbsp;</td>
-		<td width="80%" class="value"><input type="radio" name="trackingCode" id="trackingCode-analytics" value="analytics" {if $trackingCode eq "analytics"}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.analytics"}</td>
+		<td width="20%" class="label">{fieldLabel name="enableSite" key="plugins.generic.googleAnalytics.manager.settings.enableSite"}</td>
+		<td width="80%" class="value">
+			<div>
+				<span class="instruct">{translate key="plugins.generic.googleAnalytics.manager.settings.enableSiteInstructions"}</span>
+			</div>
+			<div><input type="radio" name="enableSite" id="enableSite-true" value="{$smarty.const.GOOGLE_ANALYTICS_SITE_ENABLE}" {if $trackingCode eq GOOGLE_ANALYTICS_SITE_ENABLE}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.siteEnable"}</div>
+			<div>
+				<input type="radio" name="enableSite" id="enableSite-unchanged" value="{$smarty.const.GOOGLE_ANALYTICS_SITE_UNCHANGED}" {if $trackingCode eq GOOGLE_ANALYTICS_SITE_UNCHANGED || $trackingCode eq ""}checked="checked" {/if}/> {translate key="plugins.generic.googleAnalytics.manager.settings.siteUnchanged"}
+				<dl>
+					<dt>{translate key="plugins.generic.googleAnalytics.manager.settings.googleAnalyticsSiteId"}</dt>
```

Comment:
These should be `<label>` elements for accessibility's sake; clicking the label should activate the control.